Controllers/AdminController.cs

```diff
@@ -42,12 +42,12 @@ public class AdminController : Controller
     // with an administrator account (or at least a user who has a role that has AccessAdmin permission). If you are
     // anonymous then a login page will automatically appear. The permission check (i.e. has AccessAdmin
     // permission) will be automatic as well.
-    public ActionResult Index() => View();

```

Comment:
The `TypedRoute` expects actions to return `IActionResult` or `Task<IActionResult>` ([here](https://github.com/Lombiq/Orchard-Training-Demo-Module/pull/77/files#diff-3bce31d8702cda872d4bc0164fd7ba96b4e616601348d98c004fbc3058d17522R74)). There isn't an advantage to returning `ActionResult` here so just changed it rather than make the expression more complicated.